### ISSUE TITLE
No space between yield and next token when using async function

### ISSUE THREAD

**Issue by Heliodromel** [babel/babel#1581]
Hi,

I think this might be similar to #1579.

The following code :
```
async function test() {
    await 5;
}
```
Results in (when compiled with  "asyncToGenerator" option and compact true)  :
```
"use strict";let test=_asyncToGenerator(function*(){yield5;});function _asyncToGenerator(fn){return function(){var gen=fn.apply(this, arguments);return new Promise(function(resolve, reject){var callNext=step.bind(null, "next");var callThrow=step.bind(null, "throw");function step(key, arg){try{var info=gen[key](arg);var value=info.value;}catch(error) {reject(error);return;}if(info.done){resolve(value);}else {Promise.resolve(value).then(callNext, callThrow);}}callNext();});};}
```
There is no space between yield and 5.

Could you also check ?

Regards,

**Comment by sebmck** [comment]
Fixed as of 5.4.5, thanks!

**Comment by Heliodromel** [comment]
Awesome, thanks !
